src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1625,9 +1637,13 @@ private function registerLockConfiguration(array $config, ContainerBuilder $cont
                 $container->setAlias('lock.factory', new Alias('lock.'.$resourceName.'.factory', false));
                 $container->setAlias('lock', new Alias('lock.'.$resourceName, false));
                 $container->setAlias(StoreInterface::class, new Alias('lock.store', false));
+                $container->setAlias(PersistStoreInterface::class, new Alias('lock.persist_store', false));
```

Comment:
`lock.persist_store` is not defined

should deprecate `lock.store` and `lock.store.*`